app/models/ability.rb

```diff
@@ -0,0 +1,59 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    unless user.nil?
+      initialize_admin user and return if user.is_admin?
```

Comment:
da admin und representative immer wimis sind, kann man das irgendwie zusammen ausführenlassen?
quasi die funktion so:

initialize_admin user if user.is_admin?
initialize_representative user if user.is_representative?
initialize_wimi user and return if user.is_wimi?